From 8363beef204b32592b70f8740e07d044d4ae0b5a Mon Sep 17 00:00:00 2001
From: D-mo <dimitry-ishenko@users.noreply.github.com>
Date: Mon, 16 Sep 2024 03:32:40 -0400
Subject: [PATCH 2/5] fix: Remove RPATH linking (#1560)

---
 src/shell/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/shell/CMakeLists.txt b/src/shell/CMakeLists.txt
index a7c485753..927ac8722 100644
--- a/src/shell/CMakeLists.txt
+++ b/src/shell/CMakeLists.txt
@@ -90,8 +90,6 @@ else ()
 		z
 		pthread
 	)
-
-    set_target_properties(casparcg PROPERTIES INSTALL_RPATH "$ORIGIN/../lib" BUILD_WITH_INSTALL_RPATH ON)
 endif ()
 
 add_custom_target(casparcg_copy_dependencies ALL)
-- 
2.43.0

