From 56bd9319606a78f85c0b25842f682ce039896c6c Mon Sep 17 00:00:00 2001
From: Long Nguyen <nguyen.long.908132@gmail.com>
Date: Fri, 6 Feb 2026 23:26:07 +0700
Subject: [PATCH] fix: Add missing `cstdint` include (#1697)

---
 src/common/bit_depth.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/common/bit_depth.h b/src/common/bit_depth.h
index a14375d45..fa574148a 100644
--- a/src/common/bit_depth.h
+++ b/src/common/bit_depth.h
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <memory>
+#include <cstdint>
 
 namespace caspar { namespace common {
 
@@ -13,4 +14,4 @@ enum class bit_depth : uint8_t
     bit16,
 };
 
-}} // namespace caspar::common
\ No newline at end of file
+}} // namespace caspar::common
-- 
2.53.0

