From: Dimitry Ishenko <dimitry.ishenko@gmail.com>
Subject: Remove deprecated boost system component

--- server-2.5-2.5.1.orig/src/CMakeModules/Bootstrap_Linux.cmake
+++ server-2.5-2.5.1/src/CMakeModules/Bootstrap_Linux.cmake
@@ -30,7 +30,7 @@ MARK_AS_ADVANCED (CMAKE_INSTALL_PREFIX)
 if (USE_STATIC_BOOST)
 	SET (Boost_USE_STATIC_LIBS ON)
 endif()
-find_package(Boost 1.74.0 COMPONENTS system thread filesystem log_setup log locale regex date_time coroutine REQUIRED)
+find_package(Boost 1.74.0 COMPONENTS thread filesystem log_setup log locale regex date_time coroutine REQUIRED)
 find_package(FFmpeg REQUIRED)
 find_package(OpenGL REQUIRED COMPONENTS OpenGL GLX EGL)
 find_package(GLEW REQUIRED)
