From d89a2d486347a45718b24716ed64ce7b6e2c3c7b Mon Sep 17 00:00:00 2001
From: Dimitry Ishenko <dimitry.ishenko@gmail.com>
Date: Tue, 10 Sep 2024 01:02:23 -0400
Subject: [PATCH 4/4] Clean up html producer make file

---
 src/modules/html/CMakeLists.txt | 31 ++++++++++++++-----------------
 1 file changed, 14 insertions(+), 17 deletions(-)

diff --git a/src/modules/html/CMakeLists.txt b/src/modules/html/CMakeLists.txt
index 7690049f7..baa31b933 100644
--- a/src/modules/html/CMakeLists.txt
+++ b/src/modules/html/CMakeLists.txt
@@ -1,33 +1,30 @@
-cmake_minimum_required (VERSION 3.16)
-project (html)
+cmake_minimum_required(VERSION 3.16)
+project(html)
 
 set(SOURCES
-		producer/html_cg_proxy.cpp
-		producer/html_producer.cpp
+    producer/html_cg_proxy.cpp
+    producer/html_cg_proxy.h
 
-		html.cpp
-)
-set(HEADERS
-		producer/html_cg_proxy.h
-		producer/html_producer.h
+    producer/html_producer.cpp
+    producer/html_producer.h
 
-		html.h
+    html.cpp
+    html.h
 )
 
-casparcg_add_module_project(html
-	SOURCES ${SOURCES} ${HEADERS}
-	INIT_FUNCTION "html::init"
-	UNINIT_FUNCTION "html::uninit"
-	CLI_INTERCEPTOR "html::intercept_command_line"
+casparcg_add_module_project(html SOURCES ${SOURCES}
+    INIT_FUNCTION   "html::init"
+    UNINIT_FUNCTION "html::uninit"
+    CLI_INTERCEPTOR "html::intercept_command_line"
 )
+target_link_libraries(html CEF::CEF)
 
+# TODO: remove when appropriate
 target_include_directories(html PRIVATE
     ..
     ../..
 )
 
-target_link_libraries(html CEF::CEF)
-
 set_target_properties(html PROPERTIES FOLDER modules)
 source_group(sources\\producer producer/*)
 source_group(sources ./*)
-- 
2.43.0

