--- a/src/modules/decklink/producer/decklink_producer.cpp
+++ b/src/modules/decklink/producer/decklink_producer.cpp
@@ -520,8 +520,8 @@ class decklink_producer : public IDeckLi
                 state_["has_signal"]             = has_signal_;

                 if (video) {
-                    state_["file/video/width"]  = video->GetWidth();
-                    state_["file/video/height"] = video->GetHeight();
+                    state_["file/video/width"]  = static_cast<int>(video->GetWidth());
+                    state_["file/video/height"] = static_cast<int>(video->GetHeight());
                 }
             }

